app/src/main/res/layout/faq_single_activity.xml

```diff
@@ -40,10 +41,12 @@
       <ScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:layout_gravity="center_horizontal"
         android:background="@color/oppiaGreyBackground"
         android:clipToPadding="false"
+        android:fillViewport="@bool/faq_single_activity_scroll_fill_view_port"
         android:overScrollMode="never"
```

Comment:
We can probably remove this attribute now ( As we are keeping the tablet landscape layout)

Reply:
Done